src/sheet.js

```diff
@@ -1,8 +1,25 @@
 
 import { StyleSheet } from 'glamor/lib/sheet'
 
-const sheet = new StyleSheet()
+export const sheet = new StyleSheet()
 sheet.inject()
 
+export const cache = []
```

Comment:
per my comments on your other PR, hash table should be more optimal: https://github.com/jxnblk/cxs/pull/33#discussion_r90685457